langchain4j-azure-open-ai/src/test/java/dev/langchain4j/model/azure/AzureOpenAiImageModelIT.java

```diff
@@ -0,0 +1,75 @@
+package dev.langchain4j.model.azure;
+
+import com.azure.ai.openai.models.ImageGenerationResponseFormat;
+import dev.langchain4j.data.image.Image;
+import dev.langchain4j.model.output.Response;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Base64;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+public class AzureOpenAiImageModelIT {
+
+    Logger logger = LoggerFactory.getLogger(AzureOpenAiImageModelIT.class);
+
+    @Test
+    void should_generate_image_with_url() {
+
+        AzureOpenAiImageModel model = AzureOpenAiImageModel.builder()
+                .endpoint(System.getenv("AZURE_OPENAI_ENDPOINT"))
+                .deploymentName(System.getenv("AZURE_OPENAI_DEPLOYMENT_NAME"))
+                .apiKey(System.getenv("AZURE_OPENAI_KEY"))
+                .logRequestsAndResponses(true)
+                .build();
+
+        Response<Image> response = model.generate("A coffee mug in Paris, France");
+
+        logger.info(response.toString());
+
+        Image image = response.content();
+        assertNotNull(image);
+        assertNotNull(image.url());
+        logger.info("The remote image is here: {}", image.url());
+
+        assertNull(image.base64());
```

Comment:
Feel free to rename `base64` (I've named it like this 😞) into `base64Data` in your branch, which is better name

Reply:
Oh, thanks! OK I'll do this.